cluster_test.go

```diff
@@ -122,25 +123,24 @@ rules:
 	if c.Options.ControllerManager.ExtraEnvvar["env1"] != "val1" {
 		t.Error(`c.Options.ControllerManager.ExtraEnvvar["env1"] != "val1"`)
 	}
-	kubeSchedulerHealthz := "0.0.0.0"
-	kubeSchedulerConfig, err := c.Options.Scheduler.MergeConfig(&schedulerv1beta1.KubeSchedulerConfiguration{
-		HealthzBindAddress: &kubeSchedulerHealthz,
```

Comment:
The `HealthzBindAddress` and `MetricsBindAddress` are removed.
So I use the `Parallelism` field for some tests. 

https://github.com/kubernetes/kube-scheduler/blob/v0.22.12/config/v1beta1/types.go#L56-L61
https://github.com/kubernetes/kube-scheduler/blob/v0.23.9/config/v1beta3/types.go#L42-L92